DESCRIPTION

```diff
@@ -139,5 +139,5 @@ Encoding: UTF-8
 Language: en-GB
 LazyData: true
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.2.3
+RoxygenNote: 7.3.1
 SystemRequirements: CmdStan (>=2.29)
```

Comment:
I worry about dependencies on an updated CmdStan/ CmdStanR. If users don't run an updated version they might get an esoteric error messaged.

Reply:
yeah I think this is a good point. We have an issue #400 to try and track the version of `CmdStan` for a specific version of `epinowcast` which feels like it might help?

Reply:
I thought there was issue out there. I think tracking would help for sure. I wonder if we could highjack what `CmdStanR` does right now to test for version and do a quick compare vs the feature we are using? 

See https://github.com/stan-dev/cmdstanr/blob/1f4efadaa3eaf742ebd7e60aa43806b9ae46d119/R/zzz.R#L37C1-L37C21

Issues
- [ ] currently not exported function in `CmdStanR`
- [ ] Would require some maintenance on the `epinowcast` side to ensure that VERSION >= 2.31 or whatnot

Reply:
Yeah that is a good idea. I am a little scared of trying to track minimum deps though honestly. Do you think it would be a massive headache for users to require a specific version (a soft requirement)?